README.md

```diff
@@ -38,8 +47,6 @@ Example Playbook
           nfs_enable:
             server: "{{ inventory_hostname in groups['nfs_server'] }}"
             clients: "{{ inventory_hostname in groups['nfs_clients'] }}"
-          nfs_export: "{{ hostvars['nfs_server']['nfs_export'] }}"
```

Comment:
I think both server and clients require a definition for `nfs_export`, so it makes more sense to retain this?